css-typed-om/Overview.bs

```diff
@@ -2903,6 +2903,49 @@ depending on the property they came from:
         return "currentcolor".
     2. Otherwise, return the result of serializing the <<color>> value.
 
+: 'stroke-align'
+::
+    1. Return the string representation of the keyword:
+        * center,
+        * inset,
+        * outset.
+
+: 'stroke-break'
+::
+    1. Return the string representation of the keyword:
+        * bounding-box,
+        * slice,
+        * clone.
+
+: 'stroke-linecap'
+::
+    1. Return the string representation of the keyword:
+        * butt,
+        * round,
+        * square.
+
+: 'stroke-linejoin'
+::
+    1. Let |values| initially be the empty [=list=].
+    2. If 'stroke-linejoin' has one of the following keywords,
```

Comment:
Instead list the keyword that it can't be (the one that it behaves as if omitted). This avoids us having to update this in the future if we add more keywords.  Same for step 3.